testing/upload_validation.ts

```diff
@@ -480,18 +480,126 @@ const booleanPropertySchema = zodCompleteObject<BooleanSchema & ObjectSchemaProp
   ...basePropertyValidators,
 });
 
-const numberPropertySchema = zodCompleteObject<NumberSchema & ObjectSchemaProperty>({
+const baseNumberPropertySchema = zodCompleteStrictObject<BaseNumberSchema & ObjectSchemaProperty>({
+  type: zodDiscriminant(ValueType.Number),
+  ...basePropertyValidators,
+})
+
+const numericPropertySchema = baseNumberPropertySchema.extend({
+  codaType: zodDiscriminant(ValueHintType.Percent).optional(),
+  precision: z.number().optional(),
+  useThousandsSeparator: z.boolean().optional(),
+})
+
+const scalePropertySchema = baseNumberPropertySchema.extend({
```

Comment:
I think it's unnecessary that these are extending baseNumberPropertySchema, and it's inconsistent with similar validators that just declare all of the fields without extending. In this case, I don't think you're getting anything from the base validator since you redeclare everything. So would just eliminate the extends. I made that change here:

https://github.com/coda/packs-sdk/commit/8249aa6763985634ce2c1d77d0f749bbdb7e0c3f

https://github.com/coda/packs-sdk/commit/8249aa6763985634ce2c1d77d0f749bbdb7e0c3f